src/Features/Core/Portable/Completion/Providers/AbstractPartialMethodCompletionProvider.cs

```diff
@@ -90,6 +90,7 @@ protected override async Task<ISymbol> GenerateMemberAsync(ISymbol member, IName
         if (enclosingSymbol.TypeKind is not (TypeKind.Struct or TypeKind.Class))
             return null;
 
+        // https://github.com/dotnet/roslyn/issues/73772: should we also get partial property symbols here?
```

Comment:
🙂Then we would need a new 'AbstractPartialPropertiesProvider'

Reply:
```suggestion
        // https://github.com/dotnet/roslyn/issues/73772: should we also add an `AbstractPartialPropertiesProvider`?
```